cards.ini

```diff
@@ -117,4 +117,7 @@ teamspo = Team Sponsorship
 naked man card = Adonis Campaign
 naked man = Adonis Campaign
 ddm = Deep Data Mining
-wnp = Wireless Net Pavilion
\ No newline at end of file
+wnp = Wireless Net Pavilion
+ken = Ken "Express" Tenma: Disappeared Clone
+express = Ken "Express" Tenma: Disappeared Clone
```

Comment:
Sorry for the late review!

I know that express resolves to Ken, but I'm sadly going to veto this as this is too close to "Express Delivery". I'm going to merge this PR in without the "express" part when I consolidate all the PRs together, but feel free to make a case to get it back in a separate PR!